src/main/kotlin/edu/byu/mealplanningassistant/models/Recipe.kt

```diff
@@ -1,15 +1,12 @@
 package edu.byu.mealplanningassistant.models
 
-import org.springframework.data.annotation.Id
-import org.springframework.data.relational.core.mapping.Table
 import kotlin.collections.HashSet
 
 /**
  * @author Chris West (Chris4942)
  */
 data class Recipe (
     /** Recipe ids should follow the form R_<RandomStringOfChars> and will be randomly generated by the backend */
-    val id: String?,
```

Comment:
I don't think we ever use this id so it's helpful in the DB but we don't use it so I just decided to get rid of it (and it was causing problems because mongoDB creates an ObjectId object when it sets the id and it wasn't compatible with a string so this was my lazy solution.